### PR TITLE
Using the user token to get user details

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: server/middleware/populateCurrentUser.ts [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
+      res.locals.user = {
+        ...res.locals.user,
+        userId,
+        name,
+        displayName: convertToTitleCase(name),
+        userRoles: roles.map(role => role.substring(role.indexOf('_') + 1)),
+      }
+
+      if (res.locals.user.authSource === 'nomis') {
+        res.locals.user.staffId = +userId || undefined
```
Comment (andrewrlee) [inline server/middleware/populateCurrentUser.ts:28]: any reason not to use parseInt here?

**Review by andrewrlee — DISMISSED** [review]
LGTM

**Review by petergphillips — DISMISSED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**